src/userRoutes.js

```diff
@@ -0,0 +1,58 @@
+const logger = require('../lib/log')(__filename)
+const db = require('../sequelize/db')
+const {signUp} = require('../lib/users')
+
+const obj = {}
+
+obj.createUser = async (req, res) => {
+    const userInfo = {
+        username: req.body.username,
+        email: req.body.email,
+        password: req.body.password
+    }
+    if (!userInfo.username || !userInfo.email || !userInfo.password) {
+        logger.info("invalid input")
+        return res.status(400).json({error: {message: "invalid input"}})
+    }
+    try {
+        await signUp(userInfo)
+        logger.info('Succeded creating user account', userInfo.username)
+        return res.status(200).json('Succeded creating user account')
+    } catch (err) {
+        logger.error("Creating user failed", userInfo.username, err)
+        return res.status(500).json({error: {message: 'Creating user failed. Please try again'}})
+    }
+}
+  
+obj.deleteUser = async (req, res) => {
+    if (!req.params.id) {
+        logger.info("user id was not provided")
+        return res.status(400).json({error: {message: "user id was not provided"}})
+    }
+    const {Accounts} = db.getModels()
+    try {
+        const account = await Accounts.findOne({
+        where: {
+            id: req.params.id
+        }
+        })
+        if (!account) {
+        logger.info("Cannot find user", req.params.id)
+        res.status(500).json({error: {message: "Cannot find user"}})
+        return
```

Comment:
I think we should be returning the res so our functions don't return undefined for unintended consequences

`return res.status(500).json({error: {message: "Cannot find user"}})`

Reply:
it doesn't matter